conversions/hex-bin.py

```diff
@@ -0,0 +1,44 @@
+#convert hexadecimal into binary
+from doctest import testmod
+import math
+
+
+hex_str = input("").strip()
+
+try:
+    num : int = int(hex_str, 16)
+except:
```

Comment:
Bare except clauses are not a good idea to have, instead catch the `ValueError` for the same